toolkit/scripts/update_kernel.sh

```diff
@@ -64,7 +64,7 @@ function find_old_version {
 }
 
 function update_configs {
-    CONFIG_FILE="kernel/config kernel/config_aarch64 kernel-rt/config"
+    CONFIG_FILE="kernel/config kernel/config_aarch64 kernel-rt/config kernel-hci/config"
```

Comment:
Vince, where do we apply the hci specific patches to the config file?

Reply:
The config file already contains our changes. I list the differences between it and the kernel package's config in the changelog

Reply:
thx Vince - sounds good.